### PR TITLE
fix: version-explain.md editUrl 404

### PR DESCRIPTION
**Pull request by cxhello**
Fixes: #144 

Changes: fix: version-explain.md editUrl 404

Screenshots of the change:

image

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/overview/version-explain.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 title: 版本发布说明
 keywords: [Spring Cloud Alibaba]
 description: Spring Cloud Alibaba Version.
-custom_edit_url: https://github.com/spring-cloud-alibaba-group/spring-cloud-alibaba-group.github.io/blob/main/i18n/zh-cn/docusaurus-plugin-content-docs/current/overview/terminology.md
```
[the remark this answers is not in the document]
Comment (cxhello) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/overview/version-explain.md:5]: 我测试了删除掉也没问题，有的文档也没有`custom_edit_url `这个tag.

[the remark this answers is not in the document]
Comment (yuluo-yx) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/overview/version-explain.md:5]: 可以添加一下

[the remark this answers is not in the document]
Comment (cxhello) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/overview/version-explain.md:5]: @yuluo-yx 加了貌似只会指向一个指定的url，配置文件会自动获取正确的url
https://github.com/spring-cloud-alibaba-group/spring-cloud-alibaba-group.github.io/blob/f446ab2a2c04a3e3674aa4cde0ff73152389dfe0/docusaurus.config.js#L111

[the remark this answers is not in the document]
Comment (yuluo-yx) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/overview/version-explain.md:5]: okay

**Comment by yuluo-yx** [comment]
Hi, @cxhello  This pr is causing the ci to fail, can you fix it? 

https://github.com/spring-cloud-alibaba-group/spring-cloud-alibaba-group.github.io/actions/runs/8601835648/job/23569968576

**Comment by cxhello** [comment]
@yuluo-yx I can build it locally, can you build it again?

**Comment by yuluo-yx** [comment]
> @yuluo-yx I can build it locally, can you build it again?

done!

**Comment by yuluo-yx** [comment]
@cxhello https://github.com/spring-cloud-alibaba-group/spring-cloud-alibaba-group.github.io/actions/runs/8601835648
There still seems to be a problem, you can run this command locally: `npm run build`

**Comment by cxhello** [comment]
@yuluo-yx 
image

image

**Comment by yuluo-yx** [comment]
got it. let me see that

**Comment by cxhello** [comment]
@yuluo-yx you can local test.

**Comment by yuluo-yx** [comment]
> @yuluo-yx you can local test.

got it, fixed. 🫡

**Comment by cxhello** [comment]
@cxhello website 404 
image

**Comment by cxhello** [comment]
@yuluo-yx I think may be node.js version problem.

**Pull request merged**